Source/DafnyLanguageServer.Test/Various/DiagnosticMigrationTest.cs

```diff
@@ -17,7 +17,7 @@ public class DiagnosticMigrationTest : ClientBasedLanguageServerTest {
 
   [TestMethod]
   public async Task ResolutionDiagnosticsContainPreviousVerificationResultsWhenCodeIsInsertedAfter() {
-    var documentItem = CreateTestDocument(FastToFailVerification);
+    var documentItem = CreateTestDocument(FastToFailVerification, "untitled:Untitled-1");
```

Comment:
I used this test arbitrarily because it expects one failure diagnostic which is not happening if the fix is not implemented.